UPGRADE.md

```diff
@@ -2,7 +2,19 @@
 
 ## dev-develop
 
-When upgrading also have a look at the changes in the [sulu skeleton](https://github.com/sulu/sulu-minimal/compare/2.0.0-alpha5...2.0.0-alpha6).
+When upgrading also have a look at the changes in the
+[sulu skeleton](https://github.com/sulu/sulu-minimal/compare/2.0.0-alpha5...2.0.0-alpha6).
+
+### Contact & Account API
+
+The `urls` property of the API has been renamed to `websites`, and the `socialMediaProfiles` property was renamed to
+`socialMedia`. The properties under these properties have been changed accordingly.
+
+The `emails`, `phones`, `faxes`, `websites` and `socialMedia` properties have been grouped under a `contactDetails`
+property. The types of all of these properties have changed their representation in the API. The type value is now only
```

Comment:
The `contactDetails` type representation has changed from an opject to an ID. ???